internal/cmd/load-balancer/generate-payload/generate_payload.go

```diff
@@ -0,0 +1,124 @@
+package generatepayload
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/errors"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/flags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/load-balancer/client"
+	lbUtils "github.com/stackitcloud/stackit-cli/internal/pkg/services/load-balancer/utils"
+	"github.com/stackitcloud/stackit-sdk-go/services/loadbalancer"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	instanceNameFlag = "instance-name"
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+	InstanceName *string
+}
+
+func NewCmd(p *print.Printer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "generate-payload",
+		Short: "Generates a payload to create/update a Load Balancer",
+		Long: fmt.Sprintf("%s\n%s",
+			"Generates a JSON payload with values to be used as --payload input for load balancer creation or update.",
+			"See https://docs.api.stackit.cloud/documentation/load-balancer/version/v1#tag/Load-Balancer/operation/APIService_CreateLoadBalancer for information regarding the payload structure.",
+		),
+		Args: args.NoArgs,
+		Example: examples.Build(
+			examples.NewExample(
+				`Generate a payload, and adapt it with custom values for the different configuration options`,
+				`$ stackit load-balancer generate-payload > ./payload.json`,
+				`<Modify payload in file, if needed>`,
+				`$ stackit load-balancer create --payload @./payload.json`),
+			examples.NewExample(
+				`Generate a payload with values of a load balancer, and adapt it with custom values for the different configuration options`,
+				`$ stackit load-balancer generate-payload --instance-name my-lb > ./payload.json`,
+				`<Modify payload in file>`,
+				`$ stackit load-balancer update my-lb --payload @./payload.json`),
+		),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := context.Background()
+			model, err := parseInput(p, cmd)
+			if err != nil {
+				return err
+			}
+
+			// Configure API client
+			apiClient, err := client.ConfigureClient(p)
+			if err != nil {
+				return err
+			}
+
+			var payload *loadbalancer.CreateLoadBalancerPayload
+			if model.InstanceName == nil {
+				payload = lbUtils.GetDefaultPayload()
+			} else {
+				req := buildRequest(ctx, model, apiClient)
+				resp, err := req.Execute()
+				if err != nil {
+					return fmt.Errorf("read load balancer: %w", err)
+				}
+				payload = &loadbalancer.CreateLoadBalancerPayload{
```

Comment:
As discussed before, it would be cleaner to create here an `UpdateLoadBalancerPayload`. And then create two separate functions: `outputCreateResult`, `outputUpdateResult`

Reply:
See the implementation of `argus scrape-config generate-payload` for a similar case

Reply:
Splitted the payloads